experimental/origin-admin/src/pages/marketplace/mutations/CreateListing.js

```diff
@@ -54,11 +54,13 @@ class CreateListing extends Component {
         from: seller ? seller.id : '',
         deposit: 0,
         category: props.listing.category || 'For Sale',
+        subCategory: props.listing.subCategory || '', // TODO: There's no subcategory dropdown in the modal yet
         description: props.listing.description || '',
         autoApprove: true,
         media,
         initialMedia: media,
-        unitsTotal: props.listing.unitsTotal
+        unitsTotal: props.listing.unitsTotal,
```

Comment:
Do we have an opinion on trailing commas? I noticed the linter doesn't prevent it, but we don't enforce it as well.